### DISCUSSION TITLE
Add cli usage documentation for "copilot env deploy" command

### DISCUSSION THREAD

**Discussion by tsingh2k15** [aws/copilot-cli#3871] [Ideas]
It appears that `copilot env deploy` command is not yet documented? If yes, can you please add the usage guidelines to docs under `Commands` section? If the command usage is already documented, can you please share where to see it?

I was expecting it to show under https://aws.github.io/copilot-cli/docs/commands

**Comment by paragbhingre** [comment]
@tsingh2k15 Thank you for pointing this out to us, we will add it to the doc. 
For now, you can use `copilot env deploy --help` to check the usage

**Comment by dannyrandall** [comment]
@tsingh2k15 thanks for the suggestion, it's been added to the docs page! https://aws.github.io/copilot-cli/docs/commands/env-deploy/
